### DISCUSSION TITLE
PROGRAM ABORT : Fork server handshake failed

### DISCUSSION THREAD

**Discussion by Janette88** [AFLplusplus/AFLplusplus#1349] [Q&A]
here is my problem:-)  afl++4.0.1a 

[-] Hmm, looks like the target binary terminated before we could complete a
handshake with the injected code. You can try the following:

    - The target binary crashes because necessary runtime conditions it needs
      are not met. Try to:
      1. Run again with AFL_DEBUG=1 set and check the output of the target
         binary for clues.
      2. Run again with AFL_DEBUG=1 and 'ulimit -c unlimited' and analyze the
         generated core dump.

    - Possibly the target requires a huge coverage map and has CTORS.
      Retry with setting AFL_MAP_SIZE=10000000.

Otherwise there is a horrible bug in the fuzzer.
Poke <afl-users@googlegroups.com> for troubleshooting tips.

[-] PROGRAM ABORT : Fork server handshake failed
         Location : afl_fsrv_start(), src/afl-forkserver.c:1234

how can i solve the problem? Thanks.

**Comment by vanhauser-thc** [comment]
no one can help you if you dont put any effort in describing you problem. did you try the recommendations in the error output? what was your command line? etc.

**Reply by Janette88** [reply]
so sorry to be late for the reply.  thank you for ur reminder.  i had no experience for asking the question here . i should paste my screenshot or gave more details into describing my question. i will take my effort to ask the better question :-）
 The problem had been solved .  At first, i run the command like this :
ASAN_OPTIONS=detect_leaks=0,abort_on_error=1,symbolize=0 afl-fuzz -i './afl_in' -o './afl_out' -D -t 100 -- ./app/gimp-console-2.8 --verbose -d -f @@    
  it generated the error "Fork server handshake failed" ; when  i modified 
ASAN_OPTIONS=detect_leaks=0,abort_on_error=1,symbolize=0 afl-fuzz -x ./dict/jpeg.dic -i './afl_in' -o './afl_out' -D -t 10000 -- ./app/gimp-console-2.8 --verbose -d -f @@ 
okay ,done!  it caused error last time because i  missed something  when i executed fuzzer or compiled the gimp etc....
thank you for ur reply @vanhauser-thc
